pallets/task/src/weights.rs

```diff
@@ -1,23 +1,258 @@
-use frame_support::weights::Weight;
+// This file is part of Substrate.
 
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Autogenerated weights for pallet_task
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-04-20, STEPS: `100`, REPEAT: 50, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+
+// ./target/release/node-template
+
+// benchmark
+
+// --chain
+
+// dev
+
+// --execution
+
+// wasm
+
+// --wasm-execution
+
+// compiled
+
+// --pallet
+
+// pallet_task
+
+// --extrinsic
+
+// *
+
+// --steps
+
+// 100
+
+// --repeat
+
+// 50
+
+// --output
+
+// ./pallets/task/src/weights.rs
+
+// --template
+
+// .maintain/frame-weight-template.hbs
+
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_task.
 pub trait WeightInfo {
```

Comment:
Do you have to manually create the _WeightInfo_ Trait or is autogenerated?

Reply:
in earlier version I have added it manually but after .maintain/frame-weight-template.hbs it gets auto generated.